hw/net/rocker/rocker_hw.h

```diff
@@ -165,6 +167,14 @@ enum {
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
 
+    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
+    ROCKER_TLV_CMD_TYPE_GET_PORT_STATS,
+
+    ROCKER_TLV_CMD_TYPE_P4_RMT_TABLE_ENTRY_ADD,
```

Comment:
I see.  Maybe there is a place for a comment in the code that expands the acronym?